packages/orbit-components/src/Inline/index.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import defaultTheme from "../defaultTheme";
+import { DEVICES as DEVICES_CONSTS } from "../utils/mediaQuery/consts";
+import media from "../utils/mediaQuery";
+import { normalize } from "./helpers";
+
+import type { Props } from "./index";
+
+const StyledInline = styled(({ as: Element, children }) => (
+  <Element>{children}</Element>
+)).attrs(({ dataTest }) => ({ "data-test": dataTest }))`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  // map to mediaQueries
+  ${({ viewportSizes }) => {
+    return Object.entries(viewportSizes).map(([query, value]: any) => {
+      // DEVICES_CONSTS[0] === "smallMobile"
```

Comment:
Comments should not have look like the same as the condition below them 😅 Can we rephrase it, even though that's kinda easy condition? 🙏